crates/generator/src/backend_manage.rs

```diff
@@ -11,6 +11,24 @@ pub struct Backend {
     pub validator_script_type_hash: H256,
 }
 
+impl From<Backend> for BackendInfo {
+    fn from(backend: Backend) -> Self {
```

Comment:
The `From` & `Into` normally implies light operation, but we calculate blake2b for a large binary input. I suggest we do not implement From for this conversion.